CHANGELOG.md

```diff
@@ -7,6 +7,20 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 
+## 3.17.0
+### dependencies
+* chore(deps): bump docker/setup-qemu-action from 2 to 3 by @dependabot in https://github.com/newrelic/nri-kubernetes/pull/881
+* chore(deps): bump docker/login-action from 2 to 3 by @dependabot in https://github.com/newrelic/nri-kubernetes/pull/880
+* chore(deps): bump manusa/actions-setup-minikube from 2.7.2 to 2.9.0 by @dependabot in https://github.com/newrelic/nri-kubernetes/pull/879
+* chore(deps): bump docker/setup-buildx-action from 2 to 3 by @dependabot in https://github.com/newrelic/nri-kubernetes/pull/878
+* chore(deps): update actions/checkout action to v4 by @renovate in https://github.com/newrelic/nri-kubernetes/pull/864
+* fix(deps): update kubernetes packages to v0.28.2 by @renovate in https://github.com/newrelic/nri-kubernetes/pull/861
+* chore(deps): update helm release kube-state-metrics to v5.13.0 by @renovate in https://github.com/newrelic/nri-kubernetes/pull/871
+* chore(deps): update newrelic/infrastructure-bundle docker tag to v3.2.16 by @renovate in https://github.com/newrelic/nri-kubernetes/pull/859
```

Comment:
My bad: please remove this line from the changelog and make sure my revert PR is included in this release